### PR TITLE
Remove Un-necessary pallets from Runtime

### LINKED ISSUE #229: Remove Un-necessary pallets from Runtime
# User story

Remove boilerplate pallets that came with node template.

1. Democracy
2. Lottery
3. Gilt
4. ERC721
5. Token faucet from mainnet branch

### PR DESCRIPTION
**Pull request by simonsso**
Closes #229

### CODE REVIEW

File: runtime/src/lib.rs [hunk 1]
```diff
@@ -1292,19 +1203,14 @@ construct_runtime! {
...
-        Gilt: pallet_gilt::{Pallet, Call, Storage, Event<T>, Config} = 33,
         // Pallets
         OrmlVesting: orml_vesting::{Pallet, Storage, Call, Event<T>, Config<T>} = 34,
         Currencies: orml_currencies::{Pallet, Call, Event<T>} = 35,
         Tokens: orml_tokens::{Pallet, Storage, Event<T>, Config<T>} = 36,
         PolkadexFungibleAsset: polkadex_fungible_assets::{Pallet, Call, Storage, Event<T>} = 37,
         SubstrateeRegistry: pallet_substratee_registry::{Pallet, Call, Storage, Event<T>} = 38,
         PolkadexOcex: polkadex_ocex::{Pallet, Call, Storage, Config<T>, Event<T>} = 39,
-        TokenFaucet: token_faucet_pallet::{Pallet, Call, Event<T>, Storage, ValidateUnsigned} = 40,
         ChainBridge: chainbridge::{Pallet, Call, Storage, Event<T>} = 41,
```
Comment (Gauthamastro) [inline runtime/src/lib.rs:1213]: Please remove Chainbridge too

**Review by Gauthamastro — CHANGES_REQUESTED** [review]

File: runtime/src/lib.rs [hunk 2]
```diff
@@ -1292,19 +1203,14 @@ construct_runtime! {
         Proxy: pallet_proxy::{Pallet, Call, Storage, Event<T>} = 29,
         Multisig: pallet_multisig::{Pallet, Call, Storage, Event<T>} = 30,
         Bounties: pallet_bounties::{Pallet, Call, Storage, Event<T>} = 31,
-        Lottery: pallet_lottery::{Pallet, Call, Storage, Event<T>} = 32,
-        Gilt: pallet_gilt::{Pallet, Call, Storage, Event<T>, Config} = 33,
         // Pallets
         OrmlVesting: orml_vesting::{Pallet, Storage, Call, Event<T>, Config<T>} = 34,
         Currencies: orml_currencies::{Pallet, Call, Event<T>} = 35,
         Tokens: orml_tokens::{Pallet, Storage, Event<T>, Config<T>} = 36,
         PolkadexFungibleAsset: polkadex_fungible_assets::{Pallet, Call, Storage, Event<T>} = 37,
```
Comment (Gauthamastro) [inline runtime/src/lib.rs]: Remove Polkadex FungibleAssets, SubstrateeRegistery, PolkadexOcex. IDO and Polkapool, These pallets will not be there for mainnet launch and will be upgraded via Runtime

Comment (Gauthamastro) [reply runtime/src/lib.rs]: Our focus will be on ERC20 PDEX migration and making sure staking and treasury are set and validators secure the network.

**Review by Gauthamastro — CHANGES_REQUESTED** [review]

File: runtime/src/lib.rs (hunk 1 above)
Comment (simonsso) [reply runtime/src/lib.rs]: Yes I was planning that in a separate ticket, but I will do it here

File: runtime/src/lib.rs (hunk 2 above)
Comment (simonsso) [reply runtime/src/lib.rs]: @Gauthamastro  do you know why we added this enumeration last week?

File: runtime/src/lib.rs [hunk 3]
```diff
@@ -1260,60 +1172,54 @@ construct_runtime! {
...
-        ElectionProviderMultiPhase: pallet_election_provider_multi_phase::{Pallet, Call, Storage, Event<T>, ValidateUnsigned} = 8,
-        Staking: pallet_staking::{Pallet, Call, Config<T>, Storage, Event<T>} = 9,
-        Session: pallet_session::{Pallet, Call, Storage, Event, Config<T>} = 10,
-        Democracy: pallet_democracy::{Pallet, Call, Storage, Config, Event<T>} = 11,
-        Council: pallet_collective::<Instance1>::{Pallet, Call, Storage, Origin<T>, Event<T>, Config<T>} = 12,
-        TechnicalCommittee: pallet_collective::<Instance2>::{Pallet, Call, Storage, Origin<T>, Event<T>, Config<T>} = 13,
-        Elections: pallet_elections_phragmen::{Pallet, Call, Storage, Event<T>, Config<T>} = 14,
-        TechnicalMembership: pallet_membership::<Instance1>::{Pallet, Call, Storage, Event<T>, Config<T>} = 15,
-        Grandpa: pallet_grandpa::{Pallet, Call, Storage, Config, Event, ValidateUnsigned} = 16,
-        Treasury: pallet_treasury::{Pallet, Call, Storage, Config, Event<T>} = 17,
```
Comment (Gauthamastro) [inline runtime/src/lib.rs:1280]: Please don't remove the numbering of pallets in Runtime. It's required for Ethereum contracts for PDEX migration to work

**Review by Gauthamastro — CHANGES_REQUESTED** [review]

File: runtime/src/lib.rs (hunk 2 above)
Comment (simonsso) [reply runtime/src/lib.rs]: This PR is on develop. There will be a cherry-pick to main later

File: runtime/src/lib.rs (hunk 3 above)
Comment (simonsso) [reply runtime/src/lib.rs:1280]: Re enumerate or leave holes?

Comment (Gauthamastro) [reply runtime/src/lib.rs:1280]: Re enumerate. It will break otherwise.

File: runtime/src/lib.rs (hunk 2 above)
Comment (Gauthamastro) [reply runtime/src/lib.rs]: Alright

**Review by simonsso — COMMENTED** [review]
It should be ok now.

**Review by Gauthamastro — APPROVED** [review]

**Pull request merged**